package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pxt-core",
-  "version": "6.11.1",
+  "version": "6.10.2",
```

Comment:
>6.10.2 [](start = 14, length = 6)

Intentional version downgrade?

Reply:
I'll be removing this before merging, just leaving it now in case I need to make any more changes / handle targets -- when we have changes that we need to test in targets / make pr builds for but aren't ready to merge into master, we'll occasionally bump the minor version to make a 'test branch' we can reference in builds -- in this case, I bumped master's minor branch from 6.9 to 6.11, and used 6.10 to create builds on npm for this PR, to make sure it's merge-able without breaking targets / so I could create builds like this one: https://github.com/microsoft/pxt-arcade/pull/3276